### ISSUE TITLE
Merge environment settings for SSL requests

### ISSUE THREAD

**Issue by dtoma** [core-api/python-client#152]
Hello,
Using v2.3.3, I can't make requests to an SSL endpoint:

```python
import coreapi

client = coreapi.Client()
schema = client.get('https://my-service.url/docs/:443')
```

```
requests.exceptions.SSLError: HTTPSConnectionPool(host='my-service.url', port=443): Max retries exceeded with url: /docs/ (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:645)'),))
```

According to [this issue](https://github.com/requests/requests/issues/4256) in the requests repo, environment settings need to be explicitely added to the `session.send` call in `coreapi/transports/http.py`:

```python
    def transition(self, link, decoders, params=None, link_ancestors=None, force_codec=False):
        # ...
        settings = session.merge_environment_settings(request.url, None, None, None, None)
        response = session.send(request, **settings)
        result = _decode_result(response, decoders, force_codec)
        # ...
```

This fixes the issue for me and I'm able to make requests to my service.

I have opened #153.

**Comment by tomchristie** [comment]
Closed by #153

**Comment by efagerberg** [comment]
Hello @tomchristie do you know when a version with this code will be released?
